chat-plugins/othermetas.js

```diff
@@ -0,0 +1,98 @@
+//Other metas plugin by Spandan
+'use strict';
+
+exports.commands = {
+	mixandmega: 'mnm',
+	mnm: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		if (!target || toId(target) === "" || !target.includes('@')) return this.parse('/help mixandmega');
+		let sep = target.split('@');
+		let stone = toId(sep[1]), template = toId(sep[0]);
```

Comment:
probably can use deconstructors on this 

``let [template, stone] = sep.map(toId);``  or even combine this and the line above into one line entirely